### PR TITLE
[docs] Reword some phrases

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This action installs [TestCafe](https://github.com/DevExpress/testcafe) from `np
     args: "chrome tests"
 ```
 
-In this example, the [checkout](https://github.com/actions/checkout) action checks out the repository, then `testcafe-action` installs the latest TestCafe version and runs tests from the `tests` folder in Google Chrome.
+In this example, the [checkout](https://github.com/actions/checkout) action checks out the repository. Then `testcafe-action` installs the latest TestCafe version and runs tests from the `tests` folder in Google Chrome.
```
Comment (dirk-pieterse) [inline README.md]: Why 'checks out'? What does this mean?

File: README.md [hunk 2]
```diff
@@ -44,15 +44,15 @@ The TestCafe version to install.
     args: "chrome tests"
 ```
 
-The **latest version** is installed by default.
+**Default value:** `latest` (the latest version)
```
Comment (dirk-pieterse) [inline README.md]: I'm sure 'latest' is self-explanatory and you do not need to add '(the latest version)'

File: README.md [hunk 3]
```diff
@@ -44,15 +44,15 @@ The TestCafe version to install.
...
 
 ## Examples
 
 This section contains sample workflows that show how to use `testcafe-action`.
 
 ### Run TestCafe Tests
 
-The following workflow demonstrates the basic usage of `testcafe-action`.
+The following workflow demonstrates the basic `testcafe-action` use case.
```
Comment (dirk-pieterse) [inline README.md]: "basic `testcafe-action` use case" Why 'basic use case'? Does it show hoe to use it? How it works?

**Review by dirk-pieterse — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,106 +1,106 @@
...
+- uses: DevExpress/testcafe-action@latest
+  with:
+    args: "chrome tests"
+```
+
+In this example, the [checkout](https://github.com/actions/checkout) action fetches the repository. Then `testcafe-action` installs the latest TestCafe version and runs tests from the `tests` folder in Google Chrome.
+
+The [args](#args) option specifies command line arguments passed to the [testcafe](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html) command.
+
+You can also use the [version](#version) option to specify the TestCafe version to install.
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
You can also use the [version](#version) option to specify the TestCafe version.
```

File: README.md [hunk 5]
```diff
@@ -1,106 +1,106 @@
...
+
+**Default value:** `latest`
+
+## Examples
+
+This section contains sample workflows that show how to use `testcafe-action`.
+
+### Run TestCafe Tests
+
+The following workflow demonstrates how to use `testcafe-action` in a basic scenario.
```
Comment (dirk-pieterse) [inline README.md]:
```suggestion
The following workflow demonstrates how to use `testcafe-action` in a basic scenario:
```

**Review by dirk-pieterse — APPROVED** [review]

**Review by AlexKamaev — APPROVED** [review]

**Pull request merged**